requirements/requirements-python3.8.txt

```diff
@@ -45,7 +45,7 @@ apispec==1.3.3
 appdirs==1.4.4
 argcomplete==1.11.1
 asn1crypto==1.3.0
-astroid==2.3.3
+astroid==2.4.2
```

Comment:
Have you encountered any problems with this version of the library?

Reply:
No. The file was generated when I added accept_types as dependency. Now, I have removed it. I will regenerate requirements.